library/Icinga/Authentication/PasswordPolicyHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Icinga\Authentication;
+
+use Icinga\Application\Config;
+use Icinga\Application\Hook\PasswordPolicyHook;
+use Icinga\Application\ProvidedHook\AnyPasswordPolicy;
+use Icinga\Authentication\PasswordValidator;
+
+use Icinga\Web\Form;
+
+class PasswordPolicyHelper
+{
+
+    const DEFAULT_PASSWORD_POLICY = AnyPasswordPolicy::class;
+
+    private $passwordPolicy;
```

Comment:
Please use `protected` instead of `private`. We often had situations where subclasses needed access to private variables without there being any reason why the variable was private in the first place. Therefore, the default should always be `protected` and `private` should only be used in exceptional cases and with justification.

Reply:
And it's not typed.